napari/conftest.py

```diff
@@ -757,13 +757,13 @@ def my_start(self, msec=None):
             else:
                 base_start(self)
 
-        def single_shot(msec, reciver, method=None):
+        def single_shot(msec, receiver, method=None):
```

Comment:
In general, I would prefer that we not change source code itself since this is error prone if all instances are not properly updated. Can we limit this PR to comments and docstring changes?